lib/ace/edit_session/bracket_match.js

```diff
@@ -37,6 +37,70 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.getBracketsCoor = function(position) {
+        if (position.column == 0) return null;
+
+
+        var charBeforeCursor,
+            matchLeft,
+            charAfterCursor,
+            matchRight;
+        
+        var length = -1,
+            depthLeft = 0,
+            depthRight = 0;
+
+
+        while (!matchLeft && !matchRight) {
+
+            length++;
```

Comment:
@Pochatkin ah, I see, that's because you return findClosingBracket/findOpeningBracket. Pls add an explanation comment here, like "we search in both directions until we find proper opening or closing bracket, afterwards we just find matching bracket in other direction"